js/misc/eos-launch.js

```diff
@@ -0,0 +1,66 @@
+const Gio = imports.gi.Gio;
+const GLib = imports.gi.GLib;
+
+const AppLauncherIface = '<node> \
+<interface name="org.gnome.Shell.AppLauncher"> \
+<method name="Launch"> \
+    <arg type="s" direction="in" name="name" /> \
+    <arg type="u" direction="in" name="timestamp" /> \
+    <arg type="b" direction="out" name="success" /> \
```

Comment:
I just took what we had in place before, which seems to match other methods in shellDBus.js, but I agree with you it's not needed and probably fine to change it, as it's a private thing.